docs/decisions/0010-extract-enrollment-python-api.rst

```diff
@@ -0,0 +1,65 @@
+Extract python api from POST /api/enrollment/v1/enrollment handler
+==================================================================
+
+Status
+------
+Accepted
+
+
+Decisions
+---------
+
+We will extract some of the core enrollment functionality present in the POST handler from
+    `POST /api/enrollment/v1/enrollment`
+into a non-REST Python method while leaving functionality intact.
```

Comment:
Nit: consider re-wording as "without modifying the POST handler."  Maybe also change "extract" to "copy" above.

Reply:
sure, also added a line on why we did not change post handler